apps/src/code-studio/components/pairing/StudentSelector.jsx

```diff
@@ -48,27 +70,44 @@ export default class StudentSelector extends React.Component {
       return <span>{i18n.noStudentsInSection()}</span>;
     }
 
-    const studentDivs = this.props.students.map(student => {
-      let className = 'student selectable';
-      if (this.state.selectedStudentIds.indexOf(student.id) !== -1) {
-        className = 'student selectable selected';
+    const exceededMaximum = this.state.selectedStudentIds.length >= 4;
+    const studentBtns = this.props.students.map(student => {
```

Comment:
It looks like `students` is not a required prop. Is it possible that it would be undefined? Is that a case we'd need to handle?

Reply:
Oh sorry. I just noticed this is existing code. Nevermind :)